accepted/0000-workspaces-command.md

```diff
@@ -0,0 +1,35 @@
+- Start Date: 2017-05-04
+- RFC PR:
+
+# Summary
+
+This document specify a new command, that can be used to execute subcommands inside a workspace projects.
+
+# Motivation
+
+With the addition of workspaces, it will become handy to be able to execute commands inside other projects than the one in the current directory.
```

Comment:
You can use `yarn add` to add a dependency in the current workspace, or `yarn workspace my-workspace add` to add a dependency inside a specific workspace from anywhere in the project tree.